src/Microsoft.AspNetCore.Cors/Infrastructure/CorsResult.cs

```diff
@@ -44,6 +44,11 @@ public class CorsResult
         /// </summary>
         public bool VaryByOrigin { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating if 'Simple methods' should be filtered.
```

Comment:
Even the Mozilla docs don't use the term "simple methods" (at most it says "simple requests"). But given that even those docs concede that they made the term up themselves, can we change the terminology here to be more specific and avoid made-up terms? (In both the docs and the API naming.)

Reply:
Simple requests use the 'Simple Methods' which are defined in already commited changes. I adopted the terminology from the CorsConstants.SimpleMethods variable defined already containing GET POST and HEAD HttpMethods. 

I found it to be suiting to the already in-place filter.

Reply:
Oh we already use that naming somewhere? I didn't realize that, my apologies! Perhaps this is fine then.

Reply:
Thanks @FlyingWraptor , I'll let the experts in this area provide further comments as to whether this is fitting. Based on what you've said, this does seem reasonable to me.